keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -17,17 +17,14 @@
 #include "dynamis.h"
 
 #ifdef ENCODER_ENABLE
-bool encoder_update_kb(uint8_t index, bool clockwise) {
-    if (!encoder_update_user(index, clockwise)) {
-        return false; 
+    bool encoder_update_user(uint8_t index, bool clockwise) {
```

Comment:
This won't actually work correctly, as tap_code (and tap_code16) only support basic keycodes (and modded basic keycodes). 

Ideally, this should be using the Encoder Map feature, but that's not supported by VIA yet. Though, the dynamic keymap feature (which is what powers VIA, really) actively supports it. 

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

In the meanwhile, something like this should be used:
https://github.com/qmk/qmk_firmware/blob/b66b911fa20debfb6e49756dee57d22228f5f122/keyboards/dailycraft/stickey4/keymaps/via/keymap.c#L45-L115

This allows for proper handling of all keycodes, without special cases.

Reply:
Thank you. I moved the behavior of encoder for via to keymap level code.